CurrencyConverterService/Service.cs

```diff
@@ -0,0 +1,44 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Text;
+using CurrencyConverterService.Models;
+using System.Threading;
+using System.Threading.Tasks;
+
+namespace CurrencyConverterService
+{
+    class Service
+    {
+        public static bool serviceWork = true;
+        public const int interwal = 100000;
+        public Logging logging;
+        public CurrencysRecipient currencys;
+        public CurrencyRefreshing currencyRefreshing;
+        public RatesRecipient ratesRecipient;
+        public RatesRefreshing ratesRefreshing;
+        public Service()
+        {
+            this.logging = new Logging();
```

Comment:
You should pass through constructor the _logging_ object for other service objects: CurrencysRecipient, CurrencyRefreshing, etc.